### PR TITLE
TOZ strong site code change 20 to 21  legacy update

### PR DESCRIPTION
**Pull request by salichon**
- Connections sorted 
 - 2016 Start|End date sorted for data set consistency

### CODE REVIEW

**Review by mnaguit — APPROVED** [review]
LGTM

File: install/sensors.csv [hunk 1]
```diff
@@ -556,9 +556,9 @@ Kinemetrics Inc.,FBA-ES-T,1441,MLZ,20,0,,0,0,0,0,0,0,2003-09-08T00:00:00Z,2014-0
 Kinemetrics Inc.,FBA-ES-T,1441,CRSZ,20,270,,0,0,0,0,0,0,2015-11-15T05:25:00Z,2018-08-14T23:10:00Z
 Kinemetrics Inc.,FBA-ES-T,1441,CRSZ,20,0,,0,0,0,0,0,0,2018-08-14T23:20:00Z,9999-01-01T00:00:00Z
 Kinemetrics Inc.,FBA-ES-T,145,TOZ,20,270,,0,0,0,0,0,0,1998-10-15T00:00:00Z,2008-12-04T01:45:00Z
-Kinemetrics Inc.,FBA-ES-T,145,TOZ,20,0,,0,0,0,0,0,0,2008-12-04T02:00:00Z,2016-01-12T16:00:00Z
+Kinemetrics Inc.,FBA-ES-T,145,TOZ,20,0,,0,0,0,0,0,0,2008-12-04T02:00:00Z,2016-01-13T21:00:00Z
 Kinemetrics Inc.,FBA-ES-T,164,MQZ,20,0,,0,0,0,0,0,0,1998-10-14T00:00:00Z,2016-02-23T08:00:00Z
-Kinemetrics Inc.,FBA-ES-T,1754,VRZ,20,0,,0,0,0,0,0,0,2005-02-10T03:06:00Z,2014-03-19T00:00:00Z
+`Kinemetrics Inc.,FBA-ES-T,1754,VRZ,20,0,,0,0,0,0,0,0,2005-02-10T03:06:00Z,2014-03-19T00:00:00Z
```
Comment (mnaguit) [inline install/sensors.csv:561]: remove ` ?

**Comment by mnaguit** [comment]
Travis is having a problem with  " ` " in `install//sensors.csv`
FYI @salichon

**Comment by salichon** [comment]
@mnaguit corrected a typo due to conflict resolution 
this hsould be good now.
srry 
j

**Comment by mnaguit** [comment]
No worries at all @salichon =)
Will approve now, Travis seems to be happy ;)

**Review by mnaguit — APPROVED** [review]
👍

**Pull request merged**